styles/about.css

```diff
@@ -0,0 +1,682 @@
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 0;
+  padding: 0;
+  align-items: flex-end;
+}
+
+.footer-mobile .partners-mobile {
+  display: flex;
+  text-decoration: none;
+  list-style-type: none;
+  text-align: center;
+  justify-content: center;
+  color: #d3d3d3;
+}
+
+.footer-container {
+  display: flex;
+  background: #fff;
+  color: #272a31;
+  justify-content: space-around;
+  align-items: center;
+  font-size: 10px;
+  padding: 30px;
+  text-align: center;
+}
+
+.footer-child .circle {
+  border-radius: 50%;
+  border: 3px solid #272a31;
+  color: #272a31;
+  margin-right: 3px;
+  padding: 2px;
+}
+
+.footer-child .rights {
+  font-size: 9px;
+}
+
+.footer-child .message {
+  line-height: 20px;
+  font-family: 'lato', sans-serif;
+}
+
+.footer-child {
+  padding-right: 10px;
+}
+
+.partners-mobile li {
+  width: 33%;
+}
+
+.partners-mobile .fab {
+  color: #8f8b8b;
+  font-size: 30px;
+  position: relative;
+  left: 60px;
+  top: 5px;
+}
+
+#dark,
+#lightgrey {
+  color: #b1aeae;
+  font-size: 25px;
+}
+
+/* Media screen for 560, 420, 768, 992, 1024 and above */
+
+@media only screen and (min-width: 768px) {
+  .hambuger-menu,
+  .nav__close {
+    display: none;
+  }
+
+  .nav-menu {
+    display: flex;
+  }
+
+  .main-nav {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+  }
+
+  .main-wrapper {
+    position: fixed;
+    top: 0;
+    width: 100%;
+    z-index: 5;
+  }
+
+  footer,
+  .article2,
+  .aqua-logo,
+  .past-summits,
+  .Summit-examples,
+  .images,
+  #text-logo,
+  #logo {
+    display: flex;
+  }
+
+  #main-bar {
+    text-decoration: none;
+    list-style-type: none;
+    display: flex;
+    justify-content: flex-end;
+    background-color: #272a31;
+    color: #d3d3d3;
+    width: 100%;
+    margin: 0;
+  }
+
+  .program-section li a {
+    color: #272a31;
+    font-family: 'lato', sans-serif;
+    font-size: 14px;
+  }
+
+  .program-section li a:hover {
+    color: #ec5242;
+    font-family: 'lato', sans-serif;
+    font-size: 14px;
+  }
+
+  #main-bar li a {
+    color: white;
+    font-size: 7px;
+    margin-right: 20px;
+  }
+
+  .fab {
+    font-size: 15px;
+    margin-right: 10px;
+    margin-top: 5px;
+  }
+
+  .program-section {
+    display: flex;
+    padding: 0.5% 5%;
+    align-items: center;
+    box-shadow: 2px 5px 5px rgb(39 42 49 / 50%);
+    position: static;
+    width: 90%;
+    flex-direction: row;
+    border-right: 0;
+    justify-content: space-between;
+    background: azure;
+  }
+
+  .nav-menu,
+  li {
+    display: flex;
+    text-decoration: none;
+    list-style-type: none;
+    justify-content: space-between;
+    padding-right: 5px;
+    padding-left: 5px;
+    margin: 0 3px;
+  }
+
+  .program-section img {
+    display: flex;
+    height: 50px;
+    width: 50px;
+    margin-right: 20px;
+  }
+
+  #camp {
+    border: 2px solid red;
+  }
+
+  ul.nav-menu {
+    margin: 0;
+    display: flex;
```

Comment:
* A selector class can be used instead of repeating code, keeping our code DRY is best practice and makes us write less code 👍

Reply:
man, I don't see what you want from me. absolutely nothing.